modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -47,10 +49,12 @@
 	--n-tabs-title-border-radius: initial;
 	--n-tabs-title-transition: 0.3s;
 
+  	--n-tabs-icon-align-items: center;
 	--n-tabs-icon-color: var( --e-global-color-secondary, var( --n-tabs-color-secondary-fallback ) );
 	--n-tabs-icon-color-hover: var( --n-tabs-title-color-hover );
 	--n-tabs-icon-color-active: #{$white};
 	--n-tabs-icon-size: initial;
+  	--n-tabs-svg-icon-size: 1rem;
```

Comment:
Change as requested yesterday. Please use my method.

Reply:
Changed to your method